grow_recipe/query/find_metric_value.py

```diff
@@ -30,28 +30,26 @@ def find_metric_value(xml, stage, topic, metric):
 
     xml.seek(0)
 
-    tree = etree.parse(xml)
+    tree = ElementTree.parse(xml)
+    root = tree.getroot()
 
     if not stage:
         stage = constants.DEFAULT
 
-    value = tree.xpath('/{root}/{stage}/{topic}/{metric}'
+    value = root.find('{stage}/{topic}/{metric}'
                        .format(root=constants.ROOT_NODE, stage=stage,
                                topic=topic, metric=metric))
 
-    if not value:
-        value = tree.xpath('/{root}/{stage}/{topic}/{metric}'.format(
+    if value is None:
+        value = root.find('{stage}/{topic}/{metric}'.format(
             root=constants.ROOT_NODE,
             stage=constants.DEFAULT,
             topic=topic,
             metric=metric
         ))
 
-    if not value:
+    if value is None:
         return None
 
-    # there should only be definition if the metric is present
-    assert len(value) == 1
```

Comment:
Did you still want to check for this assertion?

Reply:
It's no longer necessary because `find` only returns one result